.github/workflows/test.yml

```diff
@@ -59,11 +59,11 @@ jobs:
       # have set this up with each database as a separate job, but then we'd be
       # duplicating the matrix configuration three times.
       matrix:
-        gemfile: [ 'rails_6.1', 'rails_7.0', 'rails_7.1' ]
+        gemfile: [ 'rails_6.1', 'rails_7.0', 'rails_7.1', 'rails_7.2' ]
 
         # To keep matrix size down, only test highest and lowest rubies.
         # See "Lowest supported ruby version" in CONTRIBUTING.md
-        ruby: [ '3.0', '3.2' ]
+        ruby: [ '3.2', '3.3' ]
```

Comment:
should this be updated? 
```yml
ruby : [ '3.1', '3.3']
```
see https://endoflife.date/ruby
per https://github.com/paper-trail-gem/paper_trail/blob/master/.github/CONTRIBUTING.md#lowest-supported-ruby-version

Reply:
@kalashnikovisme looks like the following need updating as well according to [CONTRIBUTING](https://github.com/paper-trail-gem/paper_trail/blob/master/.github/CONTRIBUTING.md#lowest-supported-ruby-version): 

- https://github.com/paper-trail-gem/paper_trail/blob/be31c665c8e4a9bb07c1ce0914d5ae91a62859e4/paper_trail.gemspec#L50
- https://github.com/paper-trail-gem/paper_trail/blob/be31c665c8e4a9bb07c1ce0914d5ae91a62859e4/.rubocop.yml#L26
- https://github.com/paper-trail-gem/paper_trail/blob/be31c665c8e4a9bb07c1ce0914d5ae91a62859e4/.github/workflows/test.yml#L20

Reply:
@joshRpowell is right. This 3 files should be updated. And why have you removed 3.2?